builder/static_code.go

```diff
@@ -91,9 +113,7 @@ func ParseFile(filename string, opts ...Option) (i interface{}, err error) {
 		return nil, err
 	}
 	defer func() {
-		if closeErr := f.Close(); closeErr != nil {
-			err = closeErr
-		}
```

Comment:
This change would revert commit 7397997f31ddb24949618f11bab1588199703721, which is not the intention. Needs to be changed here (please regenerate all the generated files with `make`).

Reply:
oops, will fix, not sure how that got in there